scalafix-tests/input/src/main/scala/test/DisableSimple.scala

```diff
@@ -14,15 +14,17 @@ Disable.symbols = [
          |...
          |// scalafix:on Option.get"""
   }
-  "scala.collection.mutable.ListBuffer"
+  "scala.collection.mutable"
+]
+Disable.unlessSynthetic = [
```

Comment:
This reads weird, "disable unless synthetic" should mean this symbol is enabled/allowed when it's synthetic but the test indicates it's the opposite: symbol is disabled only if it's synthetic. Should this be `Disable.ifSynthetic` instead?